### DISCUSSION TITLE
Historical Data.. where?

### DISCUSSION THREAD

**Discussion by i3laze** [RavenSystem/esp-homekit-devices#1561] [HAA Home App]
Hi!
Got latest HAA.app update v2.0.0 and firmware v7.2.1.

Is the Historical data invoked by an “i” button at the top right corner, which does nothing? (tested this button on iPad mini 4 and iPhone 7 both latest iOS 14.7.1)

Some fresh screenshots or explanation in the AppStore are welcome.

**Comment by RavenSystem** [comment]
"i" button is the "identify" function. When "i" is pressed, accessory runs a way to identify. In HAA, status LED blinks several times.

To use Historical Data service, you must setup it: https://github.com/RavenSystem/esp-homekit-devices/wiki/Historical-Data
